spharpy/sht.py

```diff
@@ -0,0 +1,124 @@
+import numpy as np
+from pyfar import Signal
+import warnings
+from . import SphericalHarmonicSignal
+from . import SphericalHarmonics
+
+
+def sht(signal, coordinates, n_max, basis_type="real", axis=-2,
+        channel_convention='acn', normalization='n3d', condon_shortley='auto',
+        inverse_method='pseudo_inverse'):
```

Comment:
```suggestion
def sht(signal, spherical_harmonics):
```
How about passing a SphericalHarmonic object here. This simplifies things a lot.

Reply:
I think it is easier for the user to only have one method which creates the SH class and performs the transform. Moreover, with the default parameters the user only needs to pass the signal, coordinates and n_max. But I thinks its better to discuss in the weekly.

Reply:
Yes, let's discuss during the weekly.